lib/constructs/dictionary.js

```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-const conversions = require("webidl-conversions");
+const Types = require("../types.js");
```

Comment:
I'm mirroring how `require('../parameters')` is usually called `Parameters`, even though `Parameters` is not a class. See e.g. [constructs/interface.js](https://github.com/jsdom/webidl2js/blob/7a70dc7c7f90c16cbb92c880f219fea024436846/lib/constructs/interface.js#L11). Will fix the `.js`.